src/launch.py

```diff
@@ -21,22 +20,29 @@ def typer_interface(config_file: str):
 
     @staticmethod
     def launch(config):
+        crops = ["stage10.009", "stage9.009", "stage8.009", "stage7.009","stage6.009",
+                 "stage5.009", "stage4.009", "stage3.009", "stage2.009", "stage1.009", "stage0.009"]
         bpy.ops.wm.open_mainfile(filepath="src/blender_assets/CropAssets.blend")
+        # for ob in bpy.context.scene.objects:
+        #     if ob.name != "stage7.009" and ob.name != "stage10.009":
+        #         ob.select_set(True)
+        # bpy.ops.object.delete()
         for ob in bpy.context.scene.objects:
-            if ob.name != "stage11.1":
+            if ob.name not in crops:
                 ob.select_set(True)
         bpy.ops.object.delete()
         collection = "Collection"
-        cameracon = CameraController()
         cropcon = CropController(config, collection)
         scenerender = SceneRenderer(config["outfile"][0], collection)
         lightcon = LightController()
         lightcon.add_light()
-        cameracon.setup_camera("camera_one", (15, 1, 0), (1.57057, 0.00174533, 1.57057), collection)
+
         cropcon.setup_crops()
-        collection1 = bpy.data.collections.get(collection)
-        dupe = collection1.objects.get("stage11.1")
-        collection1.objects.unlink(dupe)
+        collection1 = bpy.data.collections.get("Collection")
```

Comment:
Similar comment to the one below. "Collection" should not be a hard-coded string instead, it should be the variable `collection`